docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -441,6 +441,10 @@ const registerUrl = await keycloak.createRegisterUrl();
 
 Make sure to update your code to `await` these methods.
 
+== A secure context is now required
+
+Keycloak JS now requires a link:https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts[secure context] to run properly. The reason for this is that the library now uses the Web Crypto API to calculate the SHA-256 digests needed to support PKCE. This API is only available in secure contexts, which are contexts that are served over HTTPS, `localhost` or a `.localhost` domain. If you are using the library in a non-secure context you'll need to update your development environment to use a secure context, or include a polyfill for the Web Crypto API.
```

Comment:
```suggestion
Keycloak JS now requires a link:https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts[secure context] to run. The reason for this is that the library now uses the Web Crypto API to calculate the SHA-256 digests needed to support PKCE. This API is only available in secure contexts, which are contexts that are served over HTTPS, `localhost` or a `.localhost` domain. If you are using the library in a non-secure context you'll need to update your development environment to use a secure context, or include a polyfill for the Web Crypto API.
```

Reply:
Agreed. Changed this, along with the removing the recommendation to polyfill, as this might have implications about future changes and whether they are breaking or not.